src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,155 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { getMovie } from '../../api/api';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ movies, onAdd }) => {
+  const moviePattern = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState(moviePattern);
```

Comment:
```suggestion
  const [movie, setMovie] = useState(null);
```
You just can use ```null``` for objects